package.json

```diff
@@ -30,7 +30,8 @@
     "@adobe/reactor-object-assign": "*",
     "@adobe/reactor-promise": "*",
     "@adobe/reactor-query-string": "*",
-    "@adobe/reactor-window": "*"
+    "@adobe/reactor-window": "*",
+    "lodash.isplainobject": "^4.0.6"
```

Comment:
You might want to use the https://www.npmjs.com/package/is-plain-object npm package. It minifies to about a third of the size of the lodash one and probably functions just as well.